src/audio/component.c

```diff
@@ -85,7 +84,9 @@ int comp_register(struct comp_driver *drv)
 	uint32_t flags;
 
 	irq_local_disable(flags);
-	list_item_prepend(&drv->list, &cd->list);
+	dcache_invalidate_region(drv, sizeof(*drv));
+	list_item_prepend(&drv->list, &cd.list);
+	dcache_writeback_region(drv, sizeof(*drv));
```

Comment:
This flushes the new list item added, but how about the eixsting list "cd.list". Is it safe to assume all cores have up-to-date view to it and no stale data in the cache?

Reply:
I see no flushing at all of cd overall in fact...